core/include/arch/thread_arch.h

```diff
@@ -56,6 +57,14 @@
 char *thread_arch_stack_init(thread_task_func_t task_func, void *arg, void *stack_start, int stack_size);
 
 /**
+ * @brief   interrupt usage
```

Comment:
Wait, sorry, I was confused for a moment... :-)

A good description would be something like "Get the number of bytes used on the ISR stack" or something like this.